app/README.md

```diff
@@ -0,0 +1,48 @@
+# Ethereum App Template
+
+Ethereum frontend app built with the following stack:
+
+- [Vue 3](https://v3.vuejs.org/) as the foundation
+- [Tailwind CSS](https://tailwindcss.com) for styling
```

Comment:
We may opt to replace Nightwind and Tailwind but we can leave them for now.

Reply:
Could you speak to this a bit more? Replacing the full CSS from Tailwind to some other system becomes a bigger task the longer we wait, so I think we should decide this sooner rather than later (Nightwind is just plug and play for now, so is much easier to remove later)

Reply:
Our designer @scco is big on simple design without frameworks. If we decided not to use it, it may be more of a completely remove type of thing than a replace with something else so the effort might not be as bad as you think. Our direction right now is to build just enough UI to test/try the platform that we're building so we shouldn't spend time just yet worrying about making it pretty so if we were to wipe the ui in a few weeks and start over it shouldn't mater. I'm currently in between as I have not used Tailwind before and would like to give it a shot. I also would like for it to be relatively easy for people that are not our designer to make additions/changes so a framework might be useful. On the other hand they can make things more complex. I think we can decide this over the next week or two as @scco will be devoting more time to this project a few days from now.

Reply:
Thanks for the context! Created https://github.com/dcgtc/dcgrants/issues/4 to track this